packages/cli/package.json

```diff
@@ -42,7 +42,7 @@
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
-    "prepublishOnly": "node ./scripts/rewriteShrinkwrap && npm i --package-lock-only --ignore-scripts --no-audit && npm dedupe --production",
+    "prepublishOnly": "bash ../../scripts/bundleDaemon.sh && bash ../../scripts/bundleKeytar.sh",
```

Comment:
Will this technically remove the ability for someone to create a zowe-cli.tgz locally ?
I guess we should only rely on the GH workflows, but should we have some sort of backup script equivalent to `prune-shrinkwrap`  ?